package.json

```diff
@@ -9,8 +9,7 @@
     "lint": "eslint . --ext .ts,.tsx",
     "format": "prettier --write **/*.ts",
     "docs": "rm -rf docs/ && typedoc --plugin typedoc-plugin-markdown",
-    "test": "mocha -r ts-node/register -t 120s 'tests/**/*.test.ts'",
-    "all": "npm run format && npm run lint && npm run docs && npm run build"
+    "test": "bash ./bin/runTests.sh"
```

Comment:
Why? Cant we filter on a suffix here?

Reply:
@sethvargo

Reply:
What do you mean by "filter on a suffix"?

Reply:
In auth, we inline the command, why do we need a separate bash script here?

e.g. https://github.com/google-github-actions/auth/blob/main/package.json#L10

Reply:
Thanks!

Reply:
I added a really long comment, but the short answer is "Windows". I should have done this in auth too (I'll backport eventually).